.travis.yml

```diff
@@ -2,7 +2,7 @@ language: java
 sudo: false
 
 jdk:
- - oraclejdk8
+ - openjdk11
```

Comment:
does it work without this change?

Reply:
Yeah, as I'm not using any java-11 specific features, it would work on jdk8 without any problems. However, the oraclejdk8 seems to be no longer supported on travis. Feel free to change this to something more appropriate.

Reply:
```suggestion
 - openjdk8
```